### ISSUE TITLE
How use sandbox-sse?

### ISSUE THREAD

**Issue by a269ch** [timkpaine/pyEX#112]
**How use sandbox-sse? It is example not working...**

```
c = Client(version='sandbox')
c.lastSSE(symbols='AAPL')
```

**This example doesn't work either**

```
c = Client(version='sandbox-sse')
c.lastSSE(symbols='AAPL')
```

Tell me how can I test SSE?

**Comment by timkpaine** [comment]
fixed in #113 
![Screen Shot 2020-08-04 at 14 37 03](https://user-images.githubusercontent.com/3105306/89331623-15d97a00-d660-11ea-89be-761895fb5f10.png)

**Comment by a269ch** [comment]
@timkpaine, problem #113  solved.

Commit fixed problem #113:
https://github.com/timkpaine/pyEX/commit/d7d2811602fabfced0b765feffe3782757f67fec

Thanks for the quick fix. And thanks for updating PyPI.
